images/opensuse.yaml

```diff
@@ -69,6 +69,13 @@ files:
 - path: /var/lib/dbus/machine-id
   generator: remove
 
+- path: /lib/modules-load.d/sg.conf
```

Comment:
I don't like us removing files that come from packages as those are likely to show up again in a package update.
Can we maybe figure out what processes those and instead disable or ignore failures on the systemd unit instead?

Containers in general cannot load kernel modules, so disabling the logic entirely should be fine.

Reply:
Sure, that would be `systemd-modules-load.service`